app/Chat/index.tsx

```diff
@@ -1,7 +1,130 @@
-import { Button, Group, Stack, Title, Text } from '@mantine/core';
-import { useMediaQuery } from '@mantine/hooks';
+import {
+  Button,
+  Group,
+  Stack,
+  Title,
+  Text,
+  AppShell,
+  Burger,
+  Textarea,
+  Center,
+  Avatar,
+} from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
 import { theme } from '../../components/config/theme';
+import { UserButton, SignedIn } from '@clerk/nextjs';
 
 export function Chat() {
-  return <Text>This is our chat page</Text>;
+  const [opened, { toggle }] = useDisclosure();
+
+  return (
+    <div style={{ overflowX: 'hidden' }}>
+      <div style={{ maxWidth: '100vw', position: 'relative' }}>
+        <AppShell
+          header={{ height: 60 }}
+          navbar={{
+            width: 300,
+            breakpoint: 'sm',
+            collapsed: { mobile: !opened },
+          }}
+          padding='md'
+        >
+          <AppShell.Header>
```

Comment:
do you think we need to make `header`, `nav` and `footer` components or 
since we will only have like 2 pages, are we good to copy/paste `<AppShell.Header>`, `<AppShell.Navbar>`, etc.. for each page?

Reply:
I think we could go either way! The AppShell comes with each of them in the docs on mantine, and since we just have the main chatpage I think it's fine to keep it all on one.

Reply:
perfff